Templates/GitHubActionsPipeline/.github/workflows/Feature.yml

```
@@ -47,31 +48,32 @@ jobs:
          - Setup
         runs-on: self-hosted
         steps: 
-        - name: Build git repo
-          run: ssh ${{env.zosSFTPUser}} ". ./.profile && dbbBuild.sh -w ${{ env.uniqueWorkspaceId }} -a ${{env.application}} -b ${{ env.branch }} -p ${{env.pipelineType}}"
-        - name: Prepare logs
-          if: ${{ always() }}
-          run: |
-            ssh ${{env.zosSFTPUser}} ". ./.profile && prepareLogs.sh -w ${{ env.uniqueWorkspaceId }}"  
-            rm -rf "${{ runner.temp }}/build-logs" 
-            mkdir "${{ runner.temp }}/build-logs" 
-            sftp ${{env.zosSFTPUser}}:${{ env.uniqueWorkspaceId }}/*.tar "\"${{ runner.temp }}/build-logs\""
-            tar -xvf "${{ runner.temp }}/build-logs/logs.tar" -C "${{ runner.temp }}/build-logs"
+          - name: Build git repo
+            run: ssh ${{env.zosSFTPUser}} ". ./.profile && dbbBuild.sh -w ${{ env.uniqueWorkspaceId }} -a ${{env.application}} -b ${{ env.branch }} -p ${{env.pipelineType}}"
+          - name: Prepare logs
+            if: ${{ always() }}
+            run: |
+                ssh ${{env.zosSFTPUser}} ". ./.profile && prepareLogs.sh -w ${{ env.uniqueWorkspaceId }}"  
+                rm -rf "${{ runner.temp }}/build-logs" 
+                mkdir "${{ runner.temp }}/build-logs" 
+                sftp ${{env.zosSFTPUser}}:${{ env.uniqueWorkspaceId }}/*.tar "\"${{ runner.temp }}/build-logs\""
+                tar -xvf "${{ runner.temp }}/build-logs/logs.tar" -C "${{ runner.temp }}/build-logs"
         
-        - name: Upload build logs
-          uses: actions/upload-artifact@v4
-          with:
-            name: build_${{github.run_number }}                        # Name used for artifact when it is uploaded
-            path: ${{ runner.temp }}/build-logs                        # Path to build logs artifacts
-            retention-days: 1
+          - name: Upload build logs
+            if: ${{ always() }}
+            uses: actions/upload-artifact@v4
+            with:
+                name: build_${{github.run_number }}                        # Name used for artifact when it is uploaded
+                path: ${{ runner.temp }}/build-logs                        # Path to build logs artifacts
+                retention-days: 1
 
     Package:
         needs:
          - Build
         runs-on: self-hosted
         steps:
          - name: Package build outputs
-           run: ssh ${{env.zosSFTPUser}} ". ./.profile && packageBuildOutputs.sh -w ${{ env.uniqueWorkspaceId }} -t ${{ env.application }}.tar -a ${{ env.application }} -b ${{ env.branch }} -p ${{env.pipelineType}} -v ${{ env.packageName }} "
+           run: ssh ${{env.zosSFTPUser}} ". ./.profile && packageBuildOutputs.sh -w ${{ env.uniqueWorkspaceId }} -t ${{ env.application }}.tar -a ${{ env.application }} -b ${{ env.branch }} -p ${{env.pipelineType}} -i ${{env.buildIdentifier}}"
```

The tar file reference is not required
```suggestion
           run: ssh ${{env.zosSFTPUser}} ". ./.profile && packageBuildOutputs.sh -w ${{ env.uniqueWorkspaceId }} -a ${{ env.application }} -b ${{ env.branch }} -p ${{env.pipelineType}} -i ${{env.buildIdentifier}}"
```